react/features/embed-meeting/components/EmbedMeetingTrigger.js

```diff
@@ -0,0 +1,52 @@
+// @flow
+
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { openDialog } from '../../base/dialog';
+import { translate } from '../../base/i18n';
+import { Icon, IconCodeBlock } from '../../base/icons';
+
+import EmbedMeetingDialog from './EmbedMeetingDialog';
+
+type Props = {
+
+    /**
+     * Open the embed meeting dialog
+     */
+    openEmbedDialog: Function,
+
+    /**
+     * Invoked to obtain translated strings.
+     */
+    t: Function,
+};
+
+/**
+ * Component meant to trigger showing the EmbedMeetingDialog.
+ *
+ * @returns {React$Element<any>}
+ */
+function EmbedMeetingTrigger({ t, openEmbedDialog }: Props) {
+    /**
+     * Handles opeming the embed dialog.
+     *
+     * @returns {void}
+     */
+    function onClick() {
+        openEmbedDialog(EmbedMeetingDialog);
+    }
+
+    return (
+        <div
+            className = 'embed-meeting-trigger'
+            onClick = { onClick }>
+            <Icon src = { IconCodeBlock } />
```

Comment:
All the other buttons in the invite people dialog have the icon in the right, this one stands out. Can you please double check if this is intended?

Reply:
in the provided ux, they all have the icons on the left.

Reply:
<img width="662" alt="Screenshot 2020-07-16 at 09 44 15" src="https://user-images.githubusercontent.com/1556279/87636193-09d95700-c749-11ea-8e47-8388a5bf4cba.png">